wdqs/0.3.1/whitelist.txt

```diff
@@ -0,0 +1,28 @@
+https://query.wikidata.org/sparql
+http://sparql.europeana.eu/
+http://nomisma.org/query
+http://data.cervantesvirtual.com/openrdf-sesame/repositories/data
+http://datos.bne.es/sparql
+http://edan.si.edu/saam/sparql
+http://data.bnf.fr/sparql
+http://dbpedia.org/sparql
+http://rdf.insee.fr/sparql
+http://sparql.wikipathways.org/
+http://dati.camera.it/sparql
+http://dati.emilia-romagna.it/sparql
+http://data.ordnancesurvey.co.uk/datasets/os-linked-data/apis/sparql
+http://statistics.data.gov.uk/sparql
+http://opendatacommunities.org/sparql
+http://data.plan4all.eu/sparql
+http://vocab.getty.edu/sparql.json
+http://linkeddata.uriburner.com/sparql
+http://tools.wmflabs.org/mw2sparql/sparql
+http://collection.britishart.yale.edu/sparql
+http://linkedgeodata.org/sparql
+http://sisinflab.poliba.it/semanticweb/lod/losm/sparql
+http://etna.istc.cnr.it/framester2/sparql
+http://202.45.139.84:10035/catalogs/fao/repositories/agrovoc
+https://linkeddata1.calcul.u-psud.fr/sparql
+http://sparql.hegroup.org/sparql/
+http://lov.okfn.org/dataset/lov/sparql
+http://opencitations.net/sparql
```

Comment:
Upstream has several more endpoints after this one, they should probably be added.

Reply:
Yup, I was going to do that in a followup across all 3 images once this is merged.
We could also actually pull in the whitelist used in production in the Dockerfile by wget instead of having a duplicate copy in this repo